### ISSUE TITLE
Internal Server Error when using InfluxDB 0.9 datasource?

### ISSUE THREAD

**Issue by victorhooi** [grafana/grafana#2103]
I have a Ubuntu 15.04 EC2 instance, with Grafana (v 2.1.0-pre1, commit: v2.0.2+262-g8f9eeb5, build date: 2015-05-29 16:22:46) and InfluxDB (0.9-rc31) installed.

I have added an InfluxDB 0.9 data-source into Grafana as follows:

![screenshot 2015-06-03 16 32 58](https://cloud.githubusercontent.com/assets/243499/7953943/3b526aea-0a0e-11e5-969b-cb2b968ef4d1.png)

I have then added a new graph, and I'm attempting to add data from InfluxDB - under the *Metrics* tab, I change the data-source from grafana to influxdb-local.

At the top left of the graph, I see a red exclamation mark with the error `Timeseries data request error`:

![screenshot 2015-06-03 16 30 33](https://cloud.githubusercontent.com/assets/243499/7953978/7a38062a-0a0e-11e5-92a0-9671d064e597.png)

When I go to the "FROM" field, there isn't a drop-down or auto-complete of the series in InfluxDB. On the right, I see a yellow exclamation mark with the error message `InfluxDB Error: undefined`

![screenshot 2015-06-03 16 36 36](https://cloud.githubusercontent.com/assets/243499/7954018/d33f44d6-0a0e-11e5-8abb-09affafa1842.png)

Below is what I see in `/var/log/grafana/grafana.log/` - there doesn't seem to be much else apart from `Internal Server Error`:

```
2015/06/03 06:18:50 [I] Migrator: exec migration id: create index UQE_dashboard_snapshot_delete_key - v5
2015/06/03 06:18:50 [I] Migrator: exec migration id: create index IDX_dashboard_snapshot_user_id - v5
2015/06/03 06:18:50 [I] Migrator: exec migration id: alter dashboard_snapshot to mediumtext v2
2015/06/03 06:18:50 [I] Created default admin user: admin
2015/06/03 06:18:50 [I] Listen: http://0.0.0.0:3000
2015/06/03 06:23:05 [I] Completed / 302 Found in 559.768µs
2015/06/03 06:26:45 [login.go:94 LoginPost()] [E] Invalid username or password: User not found
2015/06/03 06:26:45 [I] Completed /login 401 Unauthorized in 936.668µs
2015/06/03 06:26:45 [I] Completed /api/login/ping 401 Unauthorized in 367.953µs
2015/06/03 06:30:03 [I] ESC[1;36mCompleted /query 500 Internal Server Error in 30.122486541sESC[0m
2015/06/03 06:30:15 [I] ESC[1;36mCompleted /query 500 Internal Server Error in 30.002929832sESC[0m
2015/06/03 06:31:51 [I] Completed /logout 302 Found in 709.463µs
2015/06/03 06:32:20 [I] Completed /api/datasources 401 Unauthorized in 415.078µs
2015/06/03 06:32:20 [I] Completed /api/login/ping 401 Unauthorized in 289.532µs
2015/06/03 06:32:24 [I] Completed /datasources 302 Found in 410.126µs
2015/06/03 06:35:33 [I] ESC[1;36mCompleted /query 500 Internal Server Error in 30.121415804sESC[0m
2015/06/03 06:36:16 [I] ESC[1;36mCompleted /query 500 Internal Server Error in 30.002122372sESC[0m
2015/06/03 06:36:43 [I] ESC[1;36mCompleted /query 500 Internal Server Error in 30.002050963sESC[0m
2015/06/03 06:36:45 [I] ESC[1;36mCompleted /query 500 Internal Server Error in 30.002078856sESC[0m
2015/06/03 06:36:48 [I] ESC[1;36mCompleted /query 500 Internal Server Error in 30.002190914sESC[0m
2015/06/03 06:36:59 [I] ESC[1;36mCompleted /query 500 Internal Server Error in 30.002131692sESC[0m
```

Is there any way to find out what this `Internal Server Error` is? Or a way to get better debug logging in this case?

**Comment by torkelo** [comment]
you open chrome dev tools (f12) and inspect any response from InfluxDB (in the network tab). You can switch to "direct" access (influxdb is accessed via browser in this case), might also help troubleshoot

**Comment by victorhooi** [comment]
@torkelo Thanks for the quick reply - it was an issue on my end - sorry.

I had it set to "Proxy" mode, and I was using the FQDN for the InfluxDB address - so it was going out and coming back by the external EC2 network interface (which was blocked).

Fixed it by setting it to "Direct" - I'd already added my local workstation IP to the AWS Security Groups, so it's all good now.

Out of curiosity - is there any way to increase the log messages around this, in case I want to debug these sorts of issues on the server side?

I found the `grafana.ini` file, and I did try upping the log-level to `Trace`, however, I don't recall seeing any additional messages in `/var/log/grafana/grafana.log`?
